serving-core/src/main/scala/com/stratio/sparkta/serving/core/policy/status/PolicyStatusActor.scala

```diff
@@ -40,22 +40,32 @@ class PolicyStatusActor(curatorFramework: CuratorFramework)
     case Create(policyStatus) => sender ! create(policyStatus)
     case Update(policyStatus) => sender ! update(policyStatus)
     case FindAll => findAll
-    case Kill(name) => kill(name)
+    case Kill(name) => sender ! kill(name)
     case AddListener(name, callback) => addListener(name, callback)
   }
 
 
 
-  def kill(policyName: String): Unit = {
-    val SparkStreamingContextActorPrefix: String = "sparkStreamingContextActor"
-    implicit  val timeout:Timeout=Timeout(2L,TimeUnit.SECONDS)
-    val pActor: Option[Try[ActorRef]] = context.
-      actorSelection(s"$SparkStreamingContextActorPrefix-${policyName.replace(" ", "_")}")
-      .resolveOne().value
+  def kill(policyName: String): Boolean = {
+    implicit val timeout:Timeout = Timeout(2L, TimeUnit.SECONDS)
+    val Stopped = true
+    val NotStopped = false
+    val pActor = context.actorSelection(s"${AkkaConstant.SparkStreamingContextActor}-${policyName.replace(" ", "_")}")
+        .resolveOne().value
+
     pActor match {
-      case Some(Success(actor)) => context.stop(actor)
-      case Some(Failure(e)) => log.warn(s"there isn't $policyName actor to kill")
-      case None => log.warn(s"there isn't $policyName actor to kill")
+      case Some(Success(actor)) => {
+        context.stop(actor)
+        Stopped
+      }
+      case Some(Failure(e)) => {
+        log.warn(s"There isn't $policyName actor to kill")
```

Comment:
There isn't $policyName actor to kill -> There is no policy with name: $policyName actor to kill